CHANGELOG.md

```diff
@@ -2,6 +2,73 @@
 
 ## Unreleased:
 
+### ☢️ Breaking changes
+
+### 🌟 New javaagent instrumentation
+
+- RxJava 3
+  ([#2794](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2794))
+
+### 🌟 New library instrumentation
+
+- RxJava 3
+  ([#2794](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2794))
+
+### 📈 Improvements
+
+- URL credentials of the form `https://username:password@www.example.com/` no longer captured
+  ([#2707](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2707))
+- `@WithSpan` async support added for methods returning async Reactor 3.x types
+  ([#2714](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2714))
+- `@WithSpan` async support added for methods returning Guava ListenableFuture
+  ([#2811](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2811))
+- Semantic attributes `code.namespace` and `code.function` captured on JAX-RS internal spans
+  ([#2805](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2805))
+- `SERVER` span names improved for Spring MVC, Grails, Wicket, and Struts
+  ([#2814](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2814))
+- `SERVER` span names improved for Servlet filters
+  ([#2887](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2887))
+- `SERVER` span names improved for Resteasy
+  ([#2900](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2900))
+- `SERVER` span names improved for Jersey and CXF
+  ([#2919](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2919))
+- JAX-RS `@ApplicationPath` annotation captured as part of `SERVER` span name
+  ([#2824](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2824))
+- RequestDispatcher `forward()` and `include()` internal spans removed
+  ([#2816](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2816))
+- Context propagated to reactor-netty callbacks
+  ([#2850](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2850))
+- AWS lambda flush timeout raised to 10 seconds
```

Comment:
Let's call out significant behavior changes in a separate section when we may expect users to be expecting old behavior. Fuzzy on what that is, but I think timeouts count, span names dont